cadwyn/_asts.py

```diff
@@ -1,11 +1,15 @@
+from __future__ import annotations
```

Comment:
Let's not use `from __future__ import annotations`. I prefer to use "" for annotations that are not always available